src/node_env_var.cc

```diff
@@ -305,8 +275,12 @@ static void EnvGetter(Local<Name> property,
     return info.GetReturnValue().SetUndefined();
   }
   CHECK(property->IsString());
-  info.GetReturnValue().Set(
-      env->env_vars()->Get(env->isolate(), property.As<String>()));
+  MaybeLocal<String> value_string =
+      env->env_vars()->Get(env->isolate(), property.As<String>());
+  if (value_string.IsEmpty()) {
+    return;
+  }
+  info.GetReturnValue().Set(value_string.FromMaybe(Local<String>()));
```

Comment:
Similarly this can be just `ToLocalChecked()`

Reply:
@joyeecheung could you review the changes.